conduct.md

```diff
@@ -23,57 +23,68 @@ permission or without crediting them) such code will negatively impact
+    - Violate any part of this Code of Conduct.
+    - Violate the Allegheny honor code in regards to all submitted work.
+    - :heavy_exclamation_mark: TODO: add any additional items in the U category.
 
 #### Attendance
 
-* Acceptable Conduct (A):
-  * Members shall
-    * :heavy_exclamation_mark: TODO: add items in the A category for attendance.
+- Acceptable Conduct (A):
+
+  - Members shall
+    - :heavy_exclamation_mark: TODO: add items in the A category for attendance.
 
-* Unacceptable Conduct (U):
-  * Members shall not
-    * :heavy_exclamation_mark: TODO: add items in the U category for attendance.
+- Unacceptable Conduct (U):
+  - Members shall not
+    - :heavy_exclamation_mark: TODO: add items in the U category for attendance.
 
 #### Deadlines
 
-* Acceptable Conduct (A):
-  * Members shall
-    * :heavy_exclamation_mark: TODO: add items in the A category for deadlines.
+- Acceptable Conduct (A):
 
-* Unacceptable Conduct (U):
-  * Members shall not
-    * :heavy_exclamation_mark: TODO: add items in the U category for deadlines.
+  - Members shall
+    - Turn assignments in on time.
+    - Complete their work in a timely manner that will not negatively affect other group members.
+
+- Unacceptable Conduct (U):
+  - Members shall not
+    - Be dishonest about late assignments.
+    - Bargain for more time to complete assignments.
 
 #### Communication
 
 This section defines what communication is acceptable and unacceptable. It is vital that team members not only communicate clearly, but communicate using the right platforms and methods, such as GitHub issues tracker or slack. For example, something may be clearly communicated in Slack, but may still receive an 'unacceptable' because Slack was the inappropriate platform to use.
 
-* Acceptable Conduct (A):
-  * Members shall
-    * Use appropriate softwares for electronic communication.
-    * Always participate in the discussion.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for communication.
+- Acceptable Conduct (A):
+
+  - Members shall
+    - Use appropriate softwares for electronic communication.
+    - Always participate in the discussion.
+    - Keep all communication respectful
+    - Regularly communicate questions and issues with the class
+    - Avoid speaking over other people
+    - Give and accept constructive criticism
 
-* Unacceptable Conduct (U):
-  * Members shall not
-    * Frequently use inappropriate forums such as text messaging or using Slack instead of GitHub issues tracker.
-    * Rarely engage in the discussion.
-    * :heavy_exclamation_mark: TODO: add any additional items in the U category for communication.
+- Unacceptable Conduct (U):
+  - Members shall not
+    - Frequently use inappropriate forums such as text messaging or using Slack instead of GitHub issues tracker.
+    - Rarely engage in the discussion.
+    - Use communication platforms inappropriately.
+    - Make fun of other peoples contributions.
```

Comment:
While I definitely think that making fun of other peoples' contributions is unacceptable, I don't think this point belongs in the communication section. You could also combine it with the inappropriate use of communication platforms and identify how that looks like.